demo-terraform/aws/wordpress_ec2_alb_rds/alb.tf

```diff
@@ -1,53 +1,58 @@
 locals {
-    lb_name = "alb${random_string.lab_id.result}"
-    tg_name = "tg${random_string.lab_id.result}"
+  lb_name = "alb${random_string.lab_id.result}"
+  tg_name = "tg${random_string.lab_id.result}"
 }
 
 resource "aws_lb" "web" {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure deletion protection is enabled for AWS LB (Load Balancer)**
 
**Vulnerability**
AWS Classic load balancer has deletion protection disabled. This may lead to accidental deletion of the load balancer.
 
**Remediation**
In AWS Console - 
 1. Sign in to AWS Console and go to Amazon EMR console.
 2. Choose Create cluster.
 3. Choose Go to advanced options. 
 4. For Step 3: General Cluster Settings, under General Options make sure Deletion protection is selected to enable it.

source: accurics-